org.stellarium.Stellarium.json

```diff
@@ -13,7 +13,8 @@
         "--share=network",
         "--device=dri",
         "--socket=pulseaudio",
-        "--filesystem=home"
+        "--filesystem=~/.stellarium:create",
```

Comment:
This should be `--persist.stellarium`. However since users will have data in the wrong location it would be a pain to change it.